src/components/input-time-picker/input-time-picker.e2e.ts

```diff
@@ -559,7 +559,7 @@ describe("calcite-input-time-picker", () => {
     formAssociated("calcite-input-time-picker", { testValue: "03:23", submitsOnEnter: true });
   });
 
-  it("updates value appropriately as step changes", async () => {
+  it("updates value appropriately as step changes.", async () => {
```

Comment:
Nitpick: no need to end test names w/ periods 😉

Reply:
oh you know what, that was in there just to get the test watcher to trigger a run in watch mode, lol.